client/app/domain/anycast/domain-dns-anycast.controller.js

```diff
@@ -55,6 +57,13 @@ angular.module("App").controller(
 
         displayBC () {
             this.$scope.resetAction();
+            this.atInternet.trackOrder({
+                name: `[domain]${this.optionName}`,
+                page: "web::domain::product",
+                orderId: this.order.orderId,
```

Comment:
Those 3 lines are the same across the controllers, may be proxying them in a new service could be practical for future requests.
```
orderId: this.order.orderId,
priceTaxFree: this.order.prices.withoutTax.value,
status: 1
```